modules/imgproc/perf/perf_warp.cpp

```diff
@@ -117,44 +91,6 @@ PERF_TEST_P( TestWarpPerspective, WarpPerspective,
 #endif
 }
 
-PERF_TEST_P(TestWarpPerspective, DISABLED_WarpPerspective_ovx,
```

Comment:
OpenVX support is removed in https://github.com/opencv/opencv/pull/25197. So I remove this test. Also removed perf data from `$opencv_extra/testdata/perf/imgproc.xml`.